irc_channel.c

```diff
@@ -625,6 +625,24 @@ static gboolean control_channel_invite( irc_channel_t *ic, irc_user_t *iu )
 	return TRUE;
 }
 
+static void control_channel_kick( irc_channel_t *ic, irc_user_t *iu )
+{
+	struct irc_control_channel *icc = ic->data;
+	bee_user_t *bu = iu->bu;
+	
+	if( bu == NULL )
+		return;
+	
+	if( icc->type != IRC_CC_TYPE_GROUP )
+	{
+		irc_send_num( ic->irc, 482, "%s :Kicks are only possible to fill_by=group channels", ic->name );
```

Comment:
uhhh what? why?

Few people use fill_by=group because most protocols have shitty group support and I don't see why it should do that since all it does is remove_buddy, which removes a contact from the roster entirely. (also no idea why that one takes a group parameter)